user_guide_src/source/libraries/security.rst

```diff
@@ -204,6 +204,7 @@ The order of checking the availability of the CSRF token is as follows:
 1. ``$_POST`` array
 2. HTTP header
 3. ``php://input`` (JSON request) - bear in mind that this approach is the slowest one since we have to decode JSON and then re-encode it
+4. ``php://input`` (raw body) - for PUT, PATCH, and DELETE type of requests
```

Comment:
Please add a note that shows "``php://input`` (raw body) is checked since v4.4.2."

Reply:
Done.